source/getting-started/onboarding.markdown

```diff
@@ -3,56 +3,53 @@ title: "Onboarding Home Assistant"
 
-    ![Home Assistant preparation](/images/getting-started/onboarding_preparing_01.png)
+**Be sure to save this information safely as there is no way to recover it!**
+![Set your username and password.](/images/getting-started/username.png)
 
-With Home Assistant installed, it's time to set up the basics.
+{% details If you want to restore from backup instead%}
+If you want to restore from a backup of a previous installation, select Restore from backup.
 
-We will now create the owner's account of Home Assistant. This account is an administrator account. It will always be able to change everything.
- 
-1. If you want to restore from a backup of a previous installation, select **Restore from backup**.
-    - Continue with the procedure on [restoring from backup](/common-tasks/os/#restoring-a-backup).
-    - Ignore the rest of this procedure. The following steps describe how to create a new installation, not how to restore from backup.
-2. If this is your initial installation, select **Create my smart home**.
-3. Enter a name, username, and password.
-   - Store the name, username, and password in a password manager. There is no way to recover the owner credentials.
-   - Select **Create account**.
+Continue with the procedure on restoring from backup.
+You can ignore the remaining steps listed on this page
 
-    ![Set your username and password.](/images/getting-started/username.png)
+{% enddetails %}
 
-4. Enter the location of your home.
-   - The location is used to populate settings such as time zone, unit system, and currency.
-   - It is also used for location-based information and automations: for example showing the weather-forecast, opening the shades at sunrise, or starting the vacuum when you leave the home.
-   - If you'd rather not send your location, you can choose a location far away from where you live.
-   - You can always change this information later in the settings.
+**Step 3:** Enter the location of your home.
 
-    ![Define your location.](/images/getting-started/onboarding_location.png)
+The location is used to configure the time zone, unit system, and currency. It is also used for automations that would show the weather, open the shades at sunrise, or start the vacuum when you leave the home.
 
-5. Select which information you are willing to share.
-    - Sharing is disabled by default. However, we would like to encourage you to share some of this data.
-    - This information helps us to find out which platforms we need to support and where to focus our efforts.
-    - The data is anonymized and aggregated. To see the charts we generate out of this data, take a look at our [analytics page](https://analytics.home-assistant.io/).
-   ![Share anonymized data](/images/getting-started/onboarding_share_anonymized_info.png)
+You can always change this information later in settings.
```

Comment:
All steps tells the user how to go to the next step, it seems to be missed for this location step.